spec/models/va_profile_redis/contact_information_spec.rb

```diff
@@ -4,38 +4,14 @@
 
 describe VAProfileRedis::ContactInformation do
   let(:user) { build :user, :loa3 }
-
-  let(:service) do
-    if Flipper.enabled?(:va_v3_contact_information_service)
-      VAProfile::V2::ContactInformation::Service
-    else
-      VAProfile::ContactInformation::Service
-    end
-  end
-
-  let(:contact_person_response) do
-    if Flipper.enabled?(:va_v3_contact_information_service)
-      VAProfile::V2::ContactInformation::PersonResponse
-    else
-      VAProfile::ContactInformation::PersonResponse
-    end
-  end
-
-  let(:contact_info) { VAProfileRedis::ContactInformation.for_user(user) }
-
-  let(:person) do
-    if Flipper.enabled?(:va_v3_contact_information_service)
-      build :person_v2, telephones:
-    else
-      build :person, telephones:, permissions:
-    end
-  end
-
+  Flipper.disable(:va_v3_contact_information_service)
```

Comment:
Should this be wrapped in a `before do` block so it doesn't interfere with other tests?

Reply:
This will not influence any other tests. It should be disabled.

Reply:
I thought flippers during tests are a big reason why we get flakey tests. But if you think a `before do` isn't necessary here, that's fine.